framework/src/engine/network/schema.ts

```diff
@@ -15,6 +15,7 @@
 export const customNodeInfoSchema = {
 	$id: '/nodeInfo/custom',
 	type: 'object',
+	required: ['height', 'blockVersion', 'lastBlockID', 'maxHeightPrevoted'],
```

Comment:
`legacy` is missing here? although all of the property in codec is required

Reply:
I had thought in my mind whether we should always send legacy prop everytime we apply nodeInfo coz legacy field might not update once its synced and its unnecessary to send it again and again

Reply:
for the lisk-codec, all the properties must exist, so you need to send the value anyway although in this case, it's array, so technically it's possible to skip, for consistency we can just add empty array if it doesnt exist